code/game/objects/items/storage/belt.dm

```diff
@@ -630,12 +630,15 @@
 	if(!istype(I, /obj/item/weapon/gun/pistol))
 		return ..()
 	var/obj/item/weapon/gun/pistol/gun = I
-	if(gun.current_mag)
-		return ..()
 	for(var/obj/item/ammo_magazine/mag in contents) 
 		if(!istype(gun, mag.gun_type))
 			continue
-		gun.reload(user, mag)
+		if(user.l_hand && user.r_hand)
+			gun.tactical_reload(mag, user)
+		else if (gun.current_mag)
+			gun.tactical_reload(mag, user)
```

Comment:
make that in one big if and then early return

Reply:
would that be:
if((user.l_hand && user.r_hand) || gun.current_mag)

Reply:
keeping the else so i dont have to rewrite orient2hud()